bin/start.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I essentially moved this file into `.docker/app/` and renamed it to make its intent clearer.